docs/configure.md

```diff
@@ -65,10 +65,11 @@ java -Dotel.javaagent.configuration-file=my.properties ...
 
 ## Configuration options
```

Comment:
@jackshirazi what do you think about reorganizing this content in a structure like the content below?  (I'm not sure where `OTEL_INSTRUMENTATION_RUNTIME-TELEMETRY_EMIT-EXPERIMENTAL-TELEMETRY` fits in...)

---

## Configuration options

Because the Elastic Distribution of OpenTelemetry Java is an extension of the [OpenTelemetry Java agent](https://github.com/open-telemetry/opentelemetry-java-instrumentation), it supports:

* General OpenTelemetry configuration options
* Configuration options from OpenTelemetry extensions that are included with EDOT Java
* Configuration options that are _only_ available in EDOT Java

### OpenTelemetry configuration options

EDOT Java supports all configuration options listed in the [OpenTelemetry General SDK Configuration documentation](https://opentelemetry.io/docs/languages/sdk-configuration/general/) and [OpenTelemetry Java agent documentation](https://opentelemetry.io/docs/zero-code/java/agent/configuration/).

EDOT Java uses different defaults than the OpenTelemetry Java agent for the following configuration options:

| Option | EDOT Java default | OpenTelemetry Java agent default |
|---|---|---|
| `OTEL_RESOURCE_PROVIDERS_AWS_ENABLED` |  enabled | disabled ([docs](https://opentelemetry.io/docs/zero-code/java/agent/configuration/#enable-resource-providers-that-are-disabled-by-default)) |
| `OTEL_RESOURCE_PROVIDERS_GCP_ENABLED` | enabled | disabled ([docs](https://opentelemetry.io/docs/zero-code/java/agent/configuration/#enable-resource-providers-that-are-disabled-by-default)) |

### Configuration options from OpenTelemetry extensions

EDOT Java includes several OpenTelemetry extensions from the [OpenTelemetry Java agent contrib repo](https://github.com/open-telemetry/opentelemetry-java-contrib/). These extensions offer the following additional `OTEL_` options:

| Option(s) | Extension |  Description |
|---|---|---|
| `OTEL_SERVICE_NAME` | [Resource providers](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/resource-providers) | Can be set as usual, but if not set this will be inferred when the EDOT Java agent is running in various application servers. |
| `OTEL_INFERRED_SPANS_*` | [Inferred Spans](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/inferred-spans) |  |
| `OTEL_SPAN_STACK_TRACE_MIN_DURATION` | [Span stacktrace capture](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/span-stacktrace) | in milliseconds, defaults to 5ms |
| `OTEL_INSTRUMENTATION_RUNTIME-TELEMETRY_EMIT-EXPERIMENTAL-TELEMETRY` |  | Experimental runtime metrics are enabled by default. Set otel.instrumentation.runtime-telemetry.emit-experimental-telemetry to false to disable them. |

### EDOT Java configuration options

In addition to general OpenTelemetry configuration options, there are two kinds of configuration options that are _only_ available in EDOT Java.

**Elastic-authored options that are not yet available upstream**

Additional `OTEL_` options are configuration options that Elastic plans to contribute upstream to the [OpenTelemetry Java agent](https://github.com/open-telemetry/opentelemetry-java-instrumentation) but are not yet available in the OpenTelemetry Java agent.

_Currently there are no additional `OTEL_` options that are only available in EDOT Java._

**Elastic-specific options**

`ELASTIC_OTEL_` options that are specific to Elastic and will always live in EDOT Java (in other words, they will _not_ be added upstream) including: 

| Option(s) | Extension | Description |
|---|---|---|
| `ELASTIC_OTEL_UNIVERSAL_PROFILING_INTEGRATION_*` | [Universal profiling integration](https://github.com/elastic/elastic-otel-java/tree/main/universal-profiling-integration) |  |

Reply:
Much better thank you! Do you want to apply the changes to the branch or shall I?

Reply:
I'd be happy to apply the change and push a commit after my afternoon meetings (unless you want to go ahead to get it done earlier).